enterprise/backend/src/metabase_enterprise/serialization/v2/backfill_ids.clj

```diff
@@ -27,14 +28,14 @@
         (t2/update! model (get entity pk) {:entity_id eid})))))
 
 (defn- has-entity-id? [model]
-  (:entity_id (models/properties model)))
+  (::mi/entity-id (models/properties model)))
```

Comment:
unrelated change but this is broken since we've switched to toucan2 long ago.

```
(models/properties 'Dashboard)
> #:metabase.models.interface{:timestamped? true, :entity-id true}
```

which is namespaced map, so we should use `::mi/entity-id` for selector.